apps/studio/src/mixins/TableListContextMenus.ts

```diff
@@ -62,8 +62,16 @@ export default {
             this.$root.$emit('loadTableCreate', item)
           }
         },
-
-
+        {
+          type: 'divider'
+        },
+        {
+          name: "Delete",
+          slug: 'sql-delete',
+          handler: ({ item }) => {
+            this.$root.$emit('deleteDatabaseElement', item, 'Delete') // please format last argument in a nice way because it's shown in a modal (can move to a compute in the modal, but that's not today)
```

Comment:
For emitting events, can we have the second argument be an object with all the values in it? This makes it consistent with all the other app events.

Also please add `deleteDatabaseElement` to `AppEvent.ts`

So something like:
```js
this.$root.$emit(AppEvent.deleteDatabase, { item, action: 'delete' })
```

Also it's easy to capitalize this in the modal with LoDash, we should do that:
```
 title() {
    return _.capitalize(this.action)
 }
```